tests/legacy-cli/e2e/tests/misc/third-party-decorators.ts

```diff
@@ -24,7 +26,7 @@ export default function () {
           async getCounter() { return element(by.xpath('/html/body/app-root/div/span')).getText(); }
         }
       `,
```

Comment:
Nit: Similar point here, can we indent the code under the file paths? Hopefully this doesn't fight the formatter too much.

```typescript
// ...
  .then(_ => writeMultipleFiles({
    './e2e/src/app.po.ts': `
      import { browser, by, element } from 'protractor';
      // ...
    `,
    // ...
  }));
```